src/core/internal/execinfo.d

```diff
@@ -0,0 +1,312 @@
+/**
+ * This module helps to decide whether an appropriate execinfo implementation
+ * is available in the underling C runtime or in an external library. In the
+ * latter case exactly one of the following version identifiers should be
+ * set at the time of building druntime.
+ *
+ * Possible external execinfo version IDs based on possible backtrace output
+ * formats:
+ * $(TABLE
+ * $(THEAD Version ID, Backtrace format)
+ * $(TROW $(B ExtExecinfo_BSDFmt), 0x00000000 <_D6module4funcAFZv+0x78> at module)
+ * $(TROW $(B ExtExecinfo_DarwinFmt), 1  module    0x00000000 D6module4funcAFZv + 0)
+ * $(TROW $(B ExtExecinfo_GNUFmt), module(_D6module4funcAFZv) [0x00000000] $(B or)
+ * module(_D6module4funcAFZv+0x78) [0x00000000] $(B or) module(_D6module4funcAFZv-0x78) [0x00000000])
+ * $(TROW $(B ExtExecinfo_SolarisFmt), object'symbol+offset [pc])
+ * )
+ *
+ * The code also ensures that at most one format is selected (either by automatic
+ * C runtime detection or by $(B ExtExecinfo_) version IDs) and stores the
+ * corresponding values in $(LREF BacktraceFmt).
+ *
+ * With $(LREF getMangledSymbolName) we can get the original mangled symbol name
+ * from `backtrace_symbols` output of any supported version.
+ *
+ * Copyright: Copyright Digital Mars 2019.
+ * License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Source: $(DRUNTIMESRC core/internal/_execinfo.d)
+ */
+
+module core.internal.execinfo;
+
+version (OSX)
+    version = Darwin;
+else version (iOS)
+    version = Darwin;
+else version (TVOS)
+    version = Darwin;
+else version (WatchOS)
+    version = Darwin;
+
+version (ExtExecinfo_BSDFmt)
+    version = _extExecinfo;
+else version (ExtExecinfo_DarwinFmt)
+    version = _extExecinfo;
+else version (ExtExecinfo_GNUFmt)
+    version = _extExecinfo;
+else version (ExtExecinfo_SolarisFmt)
+    version = _extExecinfo;
+
+/**
+ * Imports the appropriate execinfo module depending on the actual
+ * OS and libc version. If the libc implementation does not include
+ * any execinfo methods, it is possible to link with an external
+ * execinfo library. In the latter case one of the $(B ExtExecinfo_)
+ * version identifiers should be set and the corresponding external
+ * library should be linked with druntime.
+ */
+mixin template ImportExecinfoPOSIX()
```

Comment:
Why do you need a `mixin template` ? The three aliases should be enough.

Reply:
You're right. It's unnecessary.